### PR TITLE
feat: Update gitbook to panacea v2

### PR DESCRIPTION
**Pull request by gyuguen**
The gitbook document has also been modified with Panacea v2

### CODE REVIEW

File: .gitbook/guide/installation.md [hunk 1]
```diff
@@ -1,39 +1,29 @@
...
-## Install the `panacead` and `panaceacli`
+## Install the `panacead`
 
 Install the latest version of Panacea Core.
 
 ```bash
 git clone https://github.com/medibloc/panacea-core
 cd panacea-core
-git checkout v1.3.3
+git checkout v2.0.0-alpha.1
```
Comment (gyuguen) [inline .gitbook/guide/installation.md:16]: I'm thinking about  whether to change this version to the official version(v2.0.0)

**Comment by gyuguen** [comment]
I think it don't use this anymore. Please let me know if I can delete it. (@youngjoon-lee )
[.gitbook/guide/light-client-daemon.md](https://github.com/medibloc/panacea-core/blob/master/.gitbook/guide/light-client-daemon.md)

**Comment by youngjoon-lee** [comment]
> I think it don't use this anymore. Please let me know if I can delete it. (@youngjoon-lee )
> 
> [.gitbook/guide/light-client-daemon.md](https://github.com/medibloc/panacea-core/blob/master/.gitbook/guide/light-client-daemon.md) 

Yes. I think we can delete it. :)

**Comment by gyuguen** [comment]
> > I think it don't use this anymore. Please let me know if I can delete it. (@youngjoon-lee )
> > [.gitbook/guide/light-client-daemon.md](https://github.com/medibloc/panacea-core/blob/master/.gitbook/guide/light-client-daemon.md)
> 
> Yes. I think we can delete it. :)

Thank you for your reply. I deleted it

File: .gitbook/guide/installation.md (hunk 1 above)
Comment (youngjoon-lee) [reply .gitbook/guide/installation.md:16]: I think that `v2.0.0-alpha.1` is good. When we release next versions, we should keep in mind that this doc also should be updated.

File: .gitbook/guide/interaction-with-the-network-cli.md [hunk 2]
```diff
@@ -107,97 +115,99 @@ panaceacli query account <account_panacea>
...
 ## Query Transactions
 
-### Matching a set of tags
+### Matching a set of events
 
-You can use the transaction search command to query for transactions that match a specific set of `tags`, which are added on every transaction.
+You can use the transaction search command to query for transactions that match a specific set of `events`, which are added on every transaction.
 
-Each tag is conformed by a key-value pair in the form of `<tag>:<value>`. Tags can also be combined to query for a more specific result using the `&` symbol.
+Events are implemented in the Cosmos SDK as an alias of the ABCI `Event` type and take the form of: `{eventType}.{attributeKey}={attributeValue}`. Events can also be combined to query for a more specific result using the `&` symbol.
```
Comment (youngjoon-lee) [inline .gitbook/guide/interaction-with-the-network-cli.md:193]: :+1:

**Review by youngjoon-lee — APPROVED** [review]
LGTM! Thank you.

**Pull request merged**